src/components/animated-image/animated-image.test.ts

```diff
@@ -1,9 +1,70 @@
-import { expect, fixture, html } from '@open-wc/testing';
+import { clickOnElement } from '../../internal/test';
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+import type SlAnimatedImage from './animated-image';
 
 describe('<sl-animated-image>', () => {
   it('should render a component', async () => {
-    const el = await fixture(html` <sl-animated-image></sl-animated-image> `);
+    const animatedImage = await fixture(html` <sl-animated-image></sl-animated-image> `);
```

Comment:
Nice. I like seeing this over `el` everywhere. Makes things easier to read. Might be worth adding to #1223.